cmd/routed-eni-cni-plugin/cni.go

```diff
@@ -215,18 +215,15 @@ func add(args *skel.CmdArgs, cniTypes typeswrapper.CNITYPES, grpcClient grpcwrap
 		return errors.Wrap(err, "add command: failed to setup network")
 	}
 
-	interfaceIndex := 0
 	ips := []*current.IPConfig{
 		{
 			Version:   "4",
 			Address:   *addr,
-			Interface: &interfaceIndex,
 		},
 	}
 
 	result := &current.Result{
 		IPs:        ips,
-		Interfaces: []*current.Interface{{Name: hostVethName}},
```

Comment:
For some still unknown reason, returning the interface here breaks the CNI when using containerd. Reverting until we have root caused the issue.